.github/CONTRIBUTING.md

```diff
@@ -240,6 +302,87 @@ Once you have completed your contribution go to the bottom of the edit page to t
+   
+   As soon as you would be happy to receive a review of your work move the card here and open a Pull Request in "Draft" status. Request a review from one of the maintainers in your Pull Request or just ask in Discord for Feedback
+   
+   **add all links/content, ready for publishing**
+   
+   While your written content might be finished, the graphics, images or links to outside content or other Wiki pages might still need some fixing or contribution. 
+   
+   **Complete**
+   
+   Content pages have been finalized and are ready to publish, Pull Request closed. Page can be added "as is" in the final Wiki version
+   
+   ![image](https://user-images.githubusercontent.com/77154511/126052466-d110127f-965a-456c-bfac-d2d52a63d043.png)
+   
+   
+   **Convert to issue**
+   
+   Every card in the Kanban can be converted into an open issue. You should use this feature. We can all work much better if we see which issues are currently open and which ones are closed and it helps us all to stay organized.
+   
+   ![image](https://user-images.githubusercontent.com/77154511/126052529-ddc9f83c-9c53-46eb-9e8c-1033d42fab14.png)
+
+  
+
+### Using issues to organize the Project Flow
+   
+   Once you have converted your Project Board card into an open issue you can add some attributes to the issue that will structurize our workflow
+   
+   ![image](https://user-images.githubusercontent.com/77154511/126052539-960c80b6-9fb6-40c7-ad01-72ec068d176b.png)
+   
+   **Assignees**
+   
+   You can assign an issue to yourself if you are working alone on your content, or to another member of the Team that you think can help or needs to have a look at that issue
+   
+   ![image](https://user-images.githubusercontent.com/77154511/126052580-107f62db-0ecf-44be-b886-1ca038d3114a.png)
+
+   **Labels**
+   
+   Labels are an excellent tool to visualize an issue. They will show up in the Issues Main Overview and directly indicate what the issue is about. For content we use the "documentation" and "feature" labels. 
+   
+   ![image](https://user-images.githubusercontent.com/77154511/126052615-f810b8b5-93b7-49e2-bc8e-eb4217560243.png)
+
+   **Projects**
+   
+   Here you can assign an issue to a project. This should be already added if you habe followed the steps above by using the Project Kanban Board. But you can also manage it here.
+   
+   ![image](https://user-images.githubusercontent.com/77154511/126052633-297926a3-911b-44a8-afb9-535875215297.png)
+
+   **Milestones**
+   
+   Milestones can help you and others see if important progress has been achieved. You are free to add milestones that you may find important
+   
+   ![image](https://user-images.githubusercontent.com/77154511/126052659-eeaf2e15-157c-4904-b665-61aee4bb51eb.png)
+
+   **Linked Pull Requests**
+   
+   As soon as you have opened a pull request for the content, link the issue to this pull request. When the Pull Request gets merged or closed it will automatically also close this issue.
```

Comment:
```suggestion
   As soon as you have opened a pull request for the content, link the issue to this pull request. When the Pull Request gets merged or closed, it will automatically also close this issue.
```